File: while.py
n = int(input("Masukkan batas perulangan: "))
print("\n")
i=1

while i<=n :
    bil = int(input("Masukkan bilangan: "))
    
    if bil % 2 == 0:
        print("Bilangan " + str(bil) + " merupakan bilangan genap \n")
    else:
        print("Bilangan " + str(bil) + " merupakan bilangan ganjil \n")
    
    i+=1

File: while.cpp
#include <iostream>
using namespace std;

int main()
{
    int bil;
    int batas;
    int i=1;
    
    cout<<"Masukkan batas perulangan : "<<endl;
    cin>>batas;
    
    while (i<=batas){
    
        cout<< "Masukkan Bilangan : "<<endl;
        cin>>bil;
        cout<<endl;
        
        if (bil%2==0)
        {
            cout<<"merupakan bilangan genap"<<endl;
        }
        else
        {
            cout<<"merupakan bilangan ganjil"<<endl;
        }
    i++;
    }

    return 0;
}
